libs/gamify/badges/src/lib/badges.controller.ts

```diff
@@ -0,0 +1,64 @@
+import { ApplicationsService } from '@gamify/application';
+import { BadRequestException, Body, Controller, Delete, Get, NotFoundException, Param, ParseIntPipe, Post, Put } from '@nestjs/common';
+import { BadgesService } from './badges.service';
+import { CreateBadgeInput } from './dto/create-badge.input';
+import { UpdateBadgeInput } from './dto/update-badge.input';
+
+@Controller('badges')
+export class BadgesController {
+  constructor(private badgesService: BadgesService, private applicationsService: ApplicationsService) {}
+
+  @Post()
+  async create(@Body() createBadgeInput: CreateBadgeInput) {
```

Comment:
Useful to add return types for all methods/endpoints?
So it's better understood as to what is returned to the client.

```js
async create(@Body() createBadgeInput: CreateBadgeInput): Badge {
  ...
```